pkg/eventprocessor/impl/webhook.go

```diff
@@ -76,14 +80,162 @@ func sendWebhookRequest(ctx context.Context, url string, body interface{}) error
 	return nil
 }
 
+type chain struct {
+	ID               int64
+	Name             string
+	ContractAddr     common.Address
+	TBLDocsURL       string
+	BlockExplorerURL string
+	SupportsNFTView  bool
+}
+
+var chains = map[int64]chain{
+	1: {
+		ID:               1,
+		Name:             "Ethereum",
```

Comment:
If you remove the Name field, you can add a comment on top of each chain to indicate the name of the chain

Reply:
Yea, good idea. Its not being used anywhere.